src/H5public.h

```diff
@@ -88,6 +88,10 @@
  * For pre-releases like \c snap0. Empty string for official releases.
  */
 #define H5_VERS_SUBRELEASE "1"
+/**
+ * Short version string
+ */
+#define H5_VERS_STR "1.15.0"
```

Comment:
I think `h5vers` will maintain this in the future, but I'm guessing for this PR that this should be set to 1.14.5 or similar?

Reply:
h5vers was updated in https://github.com/HDFGroup/hdf5/pull/4453 where H5_VERS_STR was introduced.

Reply:
ok, that's good.  #4453 is in the sync, so it's adding H5_VERS_STR, but h5vers didn't run yet in the hdf5_1_14 branch to update the version.  It should be fine.